cpu/stm32_common/periph/uart.c

```diff
@@ -145,7 +145,10 @@ static inline void wait_for_tx_complete(uart_t uart)
 void uart_write(uart_t uart, const uint8_t *data, size_t len)
 {
     assert(uart < UART_NUMOF);
-
+    if (!(dev(uart)->CR1 & USART_CR1_TE)) {
+        /* If tx is not enabled don't try to send */
```

Comment:
nit-picky: I would put the comment *above* the `if` not within. Otherwise looks good to me, how do other platforms handle this, or do they need a similar update?

Reply:
Done

Reply:
Sorry, done now.  Maybe I didn't force push.

Reply:
@smlng ping